downloads/index.md

```diff
@@ -13,10 +13,7 @@ We provide several ways for you to run Julia:
 
 Plotting capabilities are provided by external packages such as
 [PyPlot.jl](https://github.com/stevengj/PyPlot.jl) and
-[Gadfly.jl](http://gadflyjl.org). You can also do much more with
-[Compose.jl](http://composejl.org), a vector graphics library for
-Julia. Look at the [plotting instructions](plotting.html) to install a
```

Comment:
No. Fixed that. Also fixed the grammar error on the next line: "all OF these ..."